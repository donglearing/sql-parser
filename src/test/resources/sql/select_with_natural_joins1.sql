select t.a, f.b from T as t natural left join F as f natural right join H as h natural full join K as k