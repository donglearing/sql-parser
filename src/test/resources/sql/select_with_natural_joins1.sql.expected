SELECT t.a, f.b FROM T AS t NATURAL LEFT JOIN F AS f NATURAL RIGHT JOIN H AS h NATURAL FULL JOIN K AS k